src/libs/OptionsListUtils.js

```diff
@@ -714,7 +708,11 @@ function getOptions(
         );
     });
 
-    let allPersonalDetailsOptions = _.map(personalDetails, (personalDetail) =>
+    // We're only picking personal details that have logins set
+    // This is a temporary fix for all the logic that's been breaking because of the new privacy changes
+    // See https://github.com/Expensify/Expensify/issues/293465 for more context
```

Comment:
Maybe add to this comment a bit explaining the placement of this block specifically–like why do we need to put it after the logic where we generate all the report options?